assets/js/lesson_3/math_operations.js

```diff
@@ -0,0 +1,14 @@
+const firstInt = +prompt('Enter first number');
```

Comment:
it's okay to use `+` but I guess `Number` is easier to read and understand. 
```suggestion
const firstInt = Number(prompt('Enter first number'));
```

Reply:
Ok.
Thanks for the feedback. 
I'll learn it for the future